Philips.CodeAnalysis.MsTestAnalyzers/TestContextAnalyzer.cs

```diff
@@ -79,14 +77,14 @@ private static void Analyze(SyntaxNodeAnalysisContext context)
 			foreach (IdentifierNameSyntax identifier in context.Node.Parent.DescendantNodes().OfType<IdentifierNameSyntax>())
 			{
 				if ((identifier.Identifier.ToString() == propName) && (identifier.Parent != property) &&
-					!(context.SemanticModel.GetSymbolInfo(identifier).Symbol is ITypeSymbol))
+					context.SemanticModel.GetSymbolInfo(identifier).Symbol is not ITypeSymbol)
 				{
 					// if we find the same identifier as the propery and it's not a type or the original instance, it's used
 					return;
 				}
 
-				if ((identifier.Identifier.ToString() == varName) && !(identifier.Parent is VariableDeclarationSyntax) &&
-					!propNodes.Contains(identifier) && !(context.SemanticModel.GetSymbolInfo(identifier).Symbol is ITypeSymbol))
+				if ((identifier.Identifier.ToString() == varName) && identifier.Parent is not VariableDeclarationSyntax &&
```

Comment:
Consider putting each condition on its own line for readability

Reply:
Fixed, and filed Issue for automation